lib/scss/components/event-card/_event-card--item-type-link.scss

```diff
@@ -0,0 +1,18 @@
+@charset 'UTF-8';
+
+.stanford-events-more-events-block {
```

Comment:
So this is targeting a view. it should live in a style sheet separate from the event-card and more toward the style sheet for the views.